src/posts/action.md

```diff
@@ -3,56 +3,56 @@ title: Take action
+- **Sign petitions**: [International AI Treaty](https://aitreaty.org), [Ban Superintelligence](https://chng.it/Djjfj2Gmpk), [Demand responsible AI](https://www.change.org/p/artificial-intelligence-time-is-running-out-for-responsible-ai-development-91f0a02c-130a-46e1-9e55-70d6b274f4df), or one of the **national petitions**: [UK](https://petition.parliament.uk/petitions/639956), [AUS](https://www.aph.gov.au/e-petitions/petition/EN5163), [NL](https://aipetitie.nl)
 
-## Inform others
+### 2. Inform others
 
-- **Share** about AI risk on social media. This website might be a good start. <!-- Once we have a intro video that should be our go to. If we are far from that we could share some official e-flyers here? https://drive.google.com/drive/u/1/folders/1c6D_i8U95FUpfrl-eR-oRNoHUf3zghOc -->
-- **Create** [articles](/learn#articles), [videos](/learn#videos) or [memes](https://twitter.com/AISafetyMemes).
-- **Talk** to people in your life about this. Answer their questions, and get them to act. Use our [counterarguments](/counterarguments) to help you be more persuasive.
-- [**Tabling**](/tabling) and [**flyering**](/flyering) are great ways to reach many people in a short amount of time.
+- **Share** about AI risk on your social media. Tag @pauseai (Twitter), or @pause_ai (Instagram) in your posts.
+- **Talk** to people in your life about AI safety. Answer their questions, and encourage them to act too. Use our [counterarguments](/counterarguments) to help you be more persuasive.
 
-## Inform politicians
+### 3. Join / support PauseAI
 
-- [**Send an email to a politician**](/email-builder): emails are completely overpowered. Try to convince your government to work towards a pause and prepare for the [summit](/summit). Here are some [**lobby tips**](/lobby-tips).
-- **Protest**: join [one of the protests](/protests) or [organize one yourself](/organizing-a-protest).
-- **Sign petitions**: [International AI Treaty](https://aitreaty.org), [Ban Superintelligence](https://chng.it/Djjfj2Gmpk), [Demand responsible AI](https://www.change.org/p/artificial-intelligence-time-is-running-out-for-responsible-ai-development-91f0a02c-130a-46e1-9e55-70d6b274f4df), or one of the **national petitions**: [UK](https://petition.parliament.uk/petitions/639956), [AUS](https://www.aph.gov.au/e-petitions/petition/EN5163), [NL](https://aipetitie.nl)
+- **Join** the [Discord](https://discord.gg/2XXWXvErfA), where most of the collaboration happens.
+- **Find** or create a [local PauseAI community](/communities).
+- **Participate** in a protest, either on your own or with your local group. Find these on [event](/events).
+- **Discuss** how you can best contribute: fill out [this form](https://airtable.com/embed/appWPTGqZmUcs3NWu/pagoxRuCai4OYJEHt/form).
```

Comment:
This is not a form to discuss how to volunteer. This is just the Join form, which is something quite different and a form that is confusing and I think it shouldn't even exist anymore 😅.

Btw I don't get this point of "discussing" how to contribute. Maybe I'd say on the joining on the point about joining the Discord that people will be able to guide them on figuring out how to help us.